popup/main.js

```diff
@@ -1,23 +1,25 @@
 document.addEventListener("DOMContentLoaded", async () => {
-  let config = (await chrome.storage.sync.get("options"))?.options ?? {};
-  const optionsDOM = document.querySelector(".options");
-  for (const [key, value] of Object.entries(config)) {
-    const option = document.createElement("div");
-    option.className = "option";
-    option.innerHTML = `
-    <div>
-      <label for="${key}">${key}</label>
-      <p class ="desc">${value.description}</p>
-    </div>
-    <input type="checkbox" id="${key}" ${value.enable ? "checked" : ""}>
+    let config = (await chrome.storage.sync.get("options"))?.options ?? {};
+    const optionsDOM = document.querySelector(".options");
+    for (const [key, value] of Object.entries(config)) {
+        const option = document.createElement("label");
+        option.innerHTML = `
+        <div style="flex: 1;">
+            <p class="title">${key}</p>
+            <p class="desc">${value.description}</p>
+        </div>
+        <div class="toggle-wrapper">
+            <input class="toggle-input" type="checkbox" id="${key}" ${value.enable ? "checked" : ""}>
```

Comment:
Can't key include `"` and spaces?

Reply:
The easiest way of preventing issues with that would be by using `option.querySelector("input").id = key`